tools/spinel-cli/SNIFFER.md

```diff
@@ -21,9 +21,8 @@ The tool has been tested on the following platforms:
 
 ```
 sudo easy_install pip
-sudo pip install ipaddress
-sudo pip install scapy
-sudo pip install pyserial
+sudo pip install --user pyserial
+sudo pip install --user ipaddress
```

Comment:
Yes, that would use the new callback mechanism to handle PROP_THREAD_ON_MESH_NETS.  Can we push that as a follow-on PR?